docs/markdown/snowplow_media_player_common_cols.md

```diff
@@ -253,3 +253,1151 @@ If the video should restart after ending.
+{% enddocs %}
+
+
+{% docs col_useragent_patch %}
+Useragent patch version
+{% enddocs %}
+
+
+{% docs col_useragent_version %}
+Full version of the useragent
+{% enddocs %}
+
+
+{% docs col_device_class %}
+Class of device e.g. phone
+{% enddocs %}
+
+
+{% docs col_agent_class %}
+Class of agent e.g. browser
+{% enddocs %}
+
+
+{% docs col_agent_name %}
+Name of agent e.g. Chrome
+{% enddocs %}
+
+
+{% docs col_agent_name_version %}
+Name and version of agent e.g. Chrome 53.0.2785.124
+{% enddocs %}
+
+
+{% docs col_agent_name_version_major %}
+Name and major version of agent e.g. Chrome 53
+{% enddocs %}
+
+
+{% docs col_agent_version %}
+Version of agent e.g. 53.0.2785.124
+{% enddocs %}
+
+
+{% docs col_agent_version_major %}
+Major version of agent e.g. 53
+{% enddocs %}
+
+
+{% docs col_device_brand %}
+Brand of device e.g. Google
+{% enddocs %}
+
+
+{% docs col_device_name %}
+Name of device e.g. Google Nexus 6
+{% enddocs %}
+
+
+{% docs col_device_version %}
+Version of device e.g. 6.0
+{% enddocs %}
+
+
+{% docs col_layout_engine_class %}
+Class of layout engine e.g. Browser
+{% enddocs %}
+
+
+{% docs col_layout_engine_name %}
+Name of layout engine e.g. Blink
+{% enddocs %}
+
+
+{% docs col_layout_engine_name_version %}
+Name and version of layout engine e.g. Blink 53.0
+{% enddocs %}
+
+
+{% docs col_layout_engine_name_version_major %}
+Name and major version of layout engine e.g. Blink 53
+{% enddocs %}
+
+
+{% docs col_layout_engine_version %}
+Version of layout engine e.g. 53.0
+{% enddocs %}
+
+
+{% docs col_layout_engine_version_major %}
+Major version of layout engine e.g. 53
+{% enddocs %}
+
+
+{% docs col_operating_system_class %}
+Class of the OS e.g. Mobile
+{% enddocs %}
+
+
+{% docs col_operating_system_name %}
+Name of the OS e.g. Android
+{% enddocs %}
+
+
+{% docs col_operating_system_name_version %}
+Name and version of the OS e.g. Android 7.0
+{% enddocs %}
+
+
+{% docs col_operating_system_version %}
+Version of the OS e.g. 7.0
+{% enddocs %}
+
+
+{% docs col_model_tstamp %}
+The current timestamp when the model processed this row.
+{% enddocs %}
+
+{% docs col_ecommerce_user_id %}
+The ecommerce user id extracted from the ecommerce user context.
+{% enddocs %}
+
+{% docs col_ecommerce_user_email %}
+The ecommerce user email, extracted from the ecommerce user context.
+{% enddocs %}
+
+{% docs col_ecommerce_user_is_guest %}
+A boolean extracted from the ecommerce user context to ascertain whether a user is a guest or not.
+{% enddocs %}
+
+{% docs col_checkout_step_number %}
+The checkout step index e.g. 1
+{% enddocs %}
+
+{% docs col_checkout_account_type %}
+The type of account that is conducting the checkout e.g. guest
+{% enddocs %}
+
+{% docs col_checkout_billing_full_address %}
+The full billing address provided at the checkout step e.g. 1 Lincoln Street
+{% enddocs %}
+
+{% docs col_checkout_billing_postcode %}
+The full billing postcode/zipcode provided at the checkout step e.g. 90210
+{% enddocs %}
```

Comment:
Many of these will not be relevant, as they are ecom speicifc. It's not a huge deal but if you could remove any obviously ecom ones that would be for the best.